packages/website/playwright.config.ts

```diff
@@ -22,6 +23,7 @@ const config: PlaywrightTestConfig = {
   webServer: {
     command: 'yarn start',
     port: 3000,
+    reuseExistingServer: !process.env.CI,
```

Comment:
Not strictly necessary for this PR, but really nice when working locally 😄.

https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests